sentences/de/light_HassLightSet.yaml

```
@@ -17,6 +17,14 @@ intents:
           - "dimme[ (<lichter>|<alle_lichter>)][ <area>][ (auf|zu)] <brightness>"
           - "[<lichter>|<alle_lichter>][ <area>][ (auf|zu)] <brightness> dimmen"
         response: brightness
+      - sentences:
+          - "[<setzen> ][die ] Helligkeit [<hier> ]auf <brightness> [dimmen]"
```

```suggestion
          - "[<setzen> ][die ]Helligkeit [<hier> ]auf <brightness>"
          - "[die ]Helligkeit [<hier> ]auf <brightness> dimmen"
```

"dimmen" is never used with "<setzen>" beforehand, so we should split into two separate sentences to avoid unnecessary and unused possible variants